packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -105,15 +104,64 @@ + (NSString * _Nullable)version {
   return nil;
 }
 
+// Expo developers: Enable the below code by running
+//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
+// before doing pod install. This will cause the controller to see if
+// the expo-launcher packager is running, and if so, use that instead of
+// the prebuild bundle.
+// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
```

Comment:
Might help to clarify that EX_DEV_LAUNCHER_URL is an environment variable. It's clearly an env var once you think about it but I think it can't hurt to make it a little more obvious.

Also maybe "prebuilt bundle" reads better than prebuild? Up to you:

```suggestion
// Expo developers: Enable the below code by running
//     export EX_DEV_LAUNCHER_URL=http://localhost:8090
// in your shell before doing pod install. This will cause the controller to see if
// the expo-launcher packager is running, and if so, use that instead of
// the prebuilt bundle.
// See the pod_target_xcconfig definition in expo-dev-launcher.podspec
```

Reply:
Done